ui/src/callWithTelemetryAndErrorHandling.ts

```diff
@@ -47,7 +47,9 @@ export async function callWithTelemetryAndErrorHandling(callbackId: string, call
 
         if (!context.suppressErrorDisplay) {
             // Always append the error to the output channel, but only 'show' the output channel for multiline errors
-            ext.outputChannel.appendLine(localize('outputError', 'Error: {0}', errorData.message));
+            if (errorData.errorType !== 'MavenExecutionError') {
```

Comment:
Sounds questionable to me although I have no context.  I would expect though to at least have reasoning in comments in the code.  Thx.